Sources/BraintreeCard/BTCard.m

```diff
@@ -91,14 +42,7 @@ - (NSDictionary *)parameters {
         p[@"cvv"] = self.cvv;
     }
     
-    if (self.cardholderName) {
-        p[@"cardholder_name"] = self.cardholderName;
-    }
-    
     NSMutableDictionary *billingAddressDictionary = [NSMutableDictionary new];
-    if ([p[@"billing_address"] isKindOfClass:[NSDictionary class]]) {
-        [billingAddressDictionary addEntriesFromDictionary:p[@"billing_address"]];
-    }
```

Comment:
This was copying over any billing address fields passed in through the `initWithParameters` initializer that we removed. Since the initializer is gone, it should be OK to remove this.